src/components/TodoContext.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { Todo } from '../types/Todo';
+import { Status } from '../services/EnumStatusFilter';
+import { useLocaleStorage } from '../hooks/useLocaleStorage';
+import { Context } from '../services/Context';
+
+type Props = {
+  children: React.ReactNode,
+};
+
+export const TodoContext = React.createContext<Context>({
+  todos: [],
+  setTodos: () => {},
+  addTodo: () => {},
+  count: 0,
+  title: '',
+  setTitle: () => {},
+  updateTodo: () => {},
+  checkedAll: () => {},
+  selTodoFilterList: Status.ALL,
+  setSelTodoFilterList: () => {},
+  deleteTodo: () => {},
+  deleteAllCompleted: () => {},
+});
```

Comment:
Please read this article about Context usage https://www.telerik.com/blogs/react-basics-how-when-use-react-context#:~:text=Context%20is%20a%20powerful%20tool%20that%20can%20be%20used%20to,directly%20related%20to%20each%20other.

Use it only where it's needed